test.properties

```diff
@@ -7882,11 +7881,7 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
             (portal.acceptance != quarantine) AND \
             (portal.release == true) AND \
             (portal.upstream != quarantine)\
-        ) OR \
-        (portal.release ~ openjdk11)
```

Comment:
this condition has these tests affected and need to notify search team.
![image](https://github.com/user-attachments/assets/3bedfe95-d541-466d-83f9-d2092fd57091)